docs/_data/config-prague-2018.yaml

```diff
@@ -0,0 +1,76 @@
+name: Portland
+
+shortcode: prague # prague australia?
+templatecode: eu # HACK
+year: 2018
+city: Prague
+area: European
+tz: CEST
+email: prague@writethedocs.org
+
+tickets:
+  community:
+    price: €500
+  corporate:
+    price: €250
+  independent:
+    price: €150
+  student:
+    price: €100
+
+sponsorship:
+  first_draft:
+    price: €500
+  second_draft:
+    price: €1000
+  publisher:
+    price: €3000
+  patron:
+    price: €6000
+  keystone:
+    price: €12000
+  job_fair:
+    price: €800
+    half: €400
+  lanyard:
+    price: TBD
+  writing_day:
+    price: TBD
+
+date: # how do we handle these? human readable would be nice.
+  main: "**September 9-11, 2018, in Prague, Czechia**"
+  tickets_live: "January 2018"
+  month: September
+  day_one:
+    date: 8
+    dotw: Saturday
+  day_two:
+    date: 9
+    dotw: Sunday
+  day_three:
+    date: 10
+    dotw: Monday
+  day_four:
+    date: 11
+    dotw: Tuesday
+
+about:
+  summary:
+    "The main presentation track takes place from **September 9-11 (Monday and Tuesday) from 10:00 to 18:00**.
+    During the main event we also run an :doc:`/conf/prague/2018/unconference`.
+
+    You can find out more information about the venue and its accessibility on our :doc:`/conf/prague/2018/venue` page."
+  venue:
+    ":doc:`Auto Klub </conf/prague/2018/venue>`"
+
+cfp:
+  url: FIXME
+  ends: FIXME
+  notification: FIXME
```

Comment:
will these dates be in, like, 'Monday, February 32nd' format? (edited some of the text around these dates so it just says 'by [DATE]' so might need to adjust, if we use a different format)

Reply:
```
cfp:
  url: FIXME
  ends: 23:59 Sunday, May 13th CEST
  notification: 23:59 Suday, June 10th CEST
```

seem ok?